code/modules/mob/camera/camera.dm

```diff
@@ -8,7 +8,7 @@
 	mouse_opacity = 0
 	see_in_dark = 7
 	invisibility = 101 // No one can see us
-
+	sight = SEE_SELF

```

Comment:
AI could use a cursor. Blob overmind, Xenobiologists, and Abductors all get one...